### PR TITLE
Add comments for roosterjs-editor-api (part3)

### PR DESCRIPTION
**Pull request by redmondbowang**

### CODE REVIEW

File: packages/roosterjs-editor-api/lib/format/toggleSubscript.ts [hunk 1]
```diff
@@ -2,7 +2,15 @@ import execFormatWithUndo from './execFormatWithUndo';
...
 
-export default function toggleSubscript(editor: Editor): void {
+/**
+ * Toggle subscript at selection
+ * If selection is collapsed, it will only affect the input after caret
+ * If selection contains only subscript text, the subscript style will be removed
+ * If selection contains only normal text, subscript style will be added to the whole selected text
+ * If selection contains both subscript and normal text, the subscript style will be removed from whole selected text
+ * @param editor The editor instance
+ */
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-api/lib/format/toggleSubscript.ts]: What if the selection contains superscript?
Same question for subscript function

Comment (redmondbowang) [reply packages/roosterjs-editor-api/lib/format/toggleSubscript.ts]: Added description for each case

File: packages/roosterjs-editor-api/lib/format/toggleSubscript.ts [hunk 2]
```diff
@@ -2,7 +2,16 @@ import execFormatWithUndo from './execFormatWithUndo';
...
 
-export default function toggleSubscript(editor: Editor): void {
+/**
+ * Toggle subscript at selection
+ * If selection is collapsed, it will only affect the input after caret
+ * If selection contains only subscript text, the subscript style will be removed
+ * If selection contains only normal text, subscript style will be added to the whole selected text
+ * If selection contains both subscript and normal text, the subscript style will be removed from whole selected text
+ * If selection contains any superscript text, the superscript style will be removed and then subscript style will be applied
+ * @param editor The editor instance
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-api/lib/format/toggleSubscript.ts]: This is not true for some browsers, e.g. IE.

Comment (redmondbowang) [reply packages/roosterjs-editor-api/lib/format/toggleSubscript.ts]: Changed.

**Pull request merged**